book/01-introduction/sections/installing.asc

```diff
@@ -1,83 +1,83 @@
-=== Installing Git
+=== Git'i Yüklemek
 
-Before you start using Git, you have to make it available on your computer.
-Even if it's already installed, it's probably a good idea to update to the latest version.
-You can either install it as a package or via another installer, or download the source code and compile it yourself.
+Git'i kullanmaya başlamadan önce onu bilgisayarınıza kurmuş olmanız gerekmektedir.
+Halihazırda yüklenmiş olsa bile son sürüm olup olmadığını kontrol etmek, değilse de son sürüme güncellemek faydalı olacaktır.
+İsterseniz paket olarak isterseniz başka bir kurucuyla ya da kaynak kodunu indirip kendiniz derleyerek Git'i yükleyebilirsiniz.
 
 [NOTE]
 ====
-This book was written using Git version *2.8.0*.
-Though most of the commands we use should work even in ancient versions of Git, some of them might not or might act slightly differently if you're using an older version.
-Since Git is quite excellent at preserving backwards compatibility, any version after 2.0 should work just fine.
+Bu kitap Git'in *2.8.0* versiyonu baz alınarak yazılmıştır.
+Kullanacağımız çoğu kod Git'in en eski versiyonlarında bile çalışacak olsa da, eğer eski sürüm kullanıyorsanız bazıları çalışmayabilir ya da beklenenden daha farklı çalışabilir.
+Git geriye dönük uyumluluğu harika bir şekilde koruduğu için 2.0'dan sonraki herhangi bir versiyon sorun çıkarmadan çalışacaktır.
 ====
 
-==== Installing on Linux
+==== Linux'a Yüklemek
 
 (((Linux, installing)))
-If you want to install the basic Git tools on Linux via a binary installer, you can generally do so through the package management tool that comes with your distribution.
-If you're on Fedora (or any closely-related RPM-based distribution, such as RHEL or CentOS), you can use `dnf`:
+Eğer Git'in temel araçlarını Linux'a ikili kurucu aracılığıyla yüklemek istiyorsanız bunu sizin sisteminizde bulunun paket yöneticisi aracıyla yapabilirsiniz.
```

Comment:
`binary installer` kismina biraz kafa yoralim. Daha yaygin bir kullanim vardir diye dusunuyorum.

Reply:
Ben şahsen binary installer'in kendisinin bile ne olduğunu bilmediğimden yorum yapmam doğru olmayacaktır. Sizin bildiğiniz uygun ve kaliteli bir karşılık varsa onu kullanalım.

Reply:
En basit haliyle, .exe ya da .msi gibi kurulum dosyalari diyebiliriz. Ne oldugu hakkinda fikir vermesi icin.

Reply:
https://python-istihza.yazbel.com/ikili_dosyalar.html

Mesela Yazbel, binary'i direkt "ikili" olarak çevirmiş. Eğer bildiğiniz kaliteli, isabetli ve uygun bir karşılık varsa onu kullanalım.

Reply:
Buradaki kullanımına uygun bir karşılık "paket kurucusu" olurdu. Teknik terimler için bizim sözlük yararlı olabilir:
https://github.com/php/doc-tr/blob/master/README.md
Sözlük dosyanın sonunda. Her ne kadar PHP belgelerine özgü karşılıklar çoksa da yine de işe yarayacağını düşünüyorum.

Reply:
@nilgun Güncelledim, teşekkürler.